python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -398,9 +397,14 @@ def execute_k8s_job(
             start_time=start_time,
             num_pods_to_wait_for=num_pods_to_wait_for,
         )
-    except (DagsterExecutionInterruptedError, Exception) as e:
+    except (BaseException, Exception) as e:
```

Comment:
This is the same as `except BaseException as e:`

(since BaseException is a parent of Exception)